wger/manager/views/workout.py

```diff
@@ -93,6 +93,13 @@ def view(request, pk):
         if i not in muscles_back:
             muscles_back.append('images/muscles/main/muscle-{0}.svg'.format(i))
 
+    for i in canonical['muscles']['frontsecondary']:
+        if i not in muscles_front:
```

Comment:
The only thing am I'm not sure about is that if a workout has exercises that train a muscle as the main and the secondary muscle, the result is a bit fuzzy with the colors. I would suggest that we only colour the secondary muscles if there are no main ones, something like this:

```python
if i not in muscles_front and i not in canonical['muscles']['front']:
    [...]
```

The same below